packages/orbit-components/src/Accordion/AccordionSection/index.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import { useAccordion } from "../AccordionContext";
+import randomID from "../../utils/randomID";
+import useBoundingRect from "../../hooks/useBoundingRect";
+import Slide from "../../utils/Slide";
+import AccordionWrapper from "../components/AccordionWrapper";
+import SectionHeader from "./components/SectionHeader";
+import SectionFooter from "./components/SectionFooter";
+import SectionContent from "./components/SectionContent";
+
+import type { Props } from "./index";
+
+const ExpandableContent = styled.div`
+  dispaly: flex;
+  flex-direction: column;
```

Comment:
`Stack` also here, you can turn off spacing, if you dont need it, but no need to create additional component here

Reply:
I've tried using Stack here but looks like Stack is blocking `expanded` prop required for Slide component to work. I can use plain div in this case, no styles needed here.